src/assets/components/Articles.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect } from "react";
+import { getArticles } from "../../utils";
+
+export default function Articles() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    getArticles().then((response) => {
+      setArticles(response.articles)
+    }, [])
+  }, [])
+
+  return (
+    <div className="article-container">
```

Comment:
Could we change these divs to something more semantic? Ensure we are considering accessibility all the way through our development 👍